src/vm/jitinterface.cpp

```diff
@@ -10914,27 +10939,7 @@ void CEEJitInfo::reserveUnwindInfo(BOOL isFunclet, BOOL isColdCode, ULONG unwind
 
```

Comment:
@janvorli Is it possible? As I understand, it seems that ``CEEJitInfo::allocUnwindInfo`` always stores the offset from image base to unwind info as ``UnwindData``.

Please let me know if there is something wrong. If it is possible, it would be better to use that scheme.